tests/conftest.py

```diff
@@ -1,7 +1,88 @@
+import json
+
 import pytest
 from click.testing import CliRunner
+from defusedxml import ElementTree as ET
 
 
 @pytest.fixture()
 def runner():
     return CliRunner()
+
+
+@pytest.fixture()
+def timdex_record_generic_full():
```

Comment:
Not sure if having the generic datacite fixtures for test_models.py is worthwhile but I’m not sure we just want to use JPAL fixtures for every test so I left them there for now